src/js/Toolbar/L.PM.Toolbar.js

```diff
@@ -563,7 +563,28 @@ const Toolbar = L.Class.extend({
     this.changeControlOrder();
     return control;
   },
-
+  controlExists(name) {
+    return Boolean(this.getButtons()[name]);
```

Comment:
let us change this to:
```suggestion
    return Boolean(this.getButton(name));
```